scripts/analysis/amwg_table.py

```diff
@@ -97,7 +98,7 @@ def amwg_table(case_name, input_ts_loc, output_loc, var_list, write_html):
     if not input_location.is_dir():
         errmsg = "Time series directory '{}' not found.  Script is exiting.".format(input_ts_loc)
         end_diag_script(errmsg)
-
+    print(f"DEBUG: location of files is {str(input_location)}")
```

Comment:
Can you comment this print statement out and add an extra comment stating that it needs to be added to a debug log?